packages/cubejs-databricks-jdbc-driver/src/DatabricksDriver.ts

```diff
@@ -589,6 +589,9 @@ export class DatabricksDriver extends JDBCDriver {
    * Returns quoted string.
    */
   protected quoteIdentifier(identifier: string): string {
+    if (identifier.startsWith('`') && identifier.endsWith('`')) {
```

Comment:
```suggestion
    if (identifier.startsWith('`') || identifier.endsWith('`')) {
```
I'm not sure it's very relevant, but I think in case of a malformed table name, we should not escape the name as well.

Reply:
It seems like an impossible situation, and also, having backticks in Databricks' table name should not be allowed by databricks itself. So I will need further investigation

Reply:
ok, so it makes sense now: if the table name includes hyphens (for example `my-table-name`) then we add in query processing, so this fix should work